app/src/main/java/org/mifos/selfserviceapp/ui/fragments/FundTransferFragment.java

```diff
@@ -0,0 +1,209 @@
+
+    @Inject
+    FundTransferPresenter mFundTransferPresenter;
+
+    private View rootView;
+    private long fromClientId;
+    private long fromOfficeId;
+    private long fromAccountId;
+    private long fromAccountType;
+    private long toClientId;
+    private long toOfficeId;
+    private long toAccountId;
+    private long toAccountType;
+
+    @BindView(R.id.tv_transfer_date)
+    TextView tvTransactionDate;
+    @BindView(R.id.et_transfer_date)
+    EditText etTransferDate;
+    @BindView(R.id.tv_transfer_description)
+    TextView tvTransferDescription;
+    @BindView(R.id.et_transfer_description)
+    EditText etTransferDescription;
+    @BindView(R.id.tv_amount_transfer)
+    TextView tvAmountTransfer;
+    @BindView(R.id.et_amount_transfer)
+    EditText etAmountTransfer;
+    @BindView(R.id.tv_from_account)
+    TextView tvFromAccount;
+    @BindView(R.id.sp_from_account)
+    Spinner spFromAccount;
+    @BindView(R.id.tv_to_account)
+    TextView tvToAccount;
+    @BindView(R.id.sp_to_account)
+    Spinner spToAccount;
+    @BindView(R.id.bt_transfer_amount)
+    Button btTransferAmount;
+    @BindView(R.id.bt_cancel_payment)
+    Button btCancelPayment;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((BaseActivity) getActivity()).getActivityComponent().inject(this);
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        rootView = inflater.inflate(R.layout.fragment_fund_transfer, container, false);
+        ButterKnife.bind(this, rootView);
+
+        mFundTransferPresenter.attachView(this);
+
+        mFundTransferPresenter.loadAccountTransferTemplate();
+
+        return rootView;
+    }
+
+    @Override
+    public void showFundTransferTemplate(final FundTransferTemplateResponse fundTransferTemplate) {
+        List<FundTransferTemplate> fromAccountNumber = fundTransferTemplate.getFromAccountOptions();
+        List<FundTransferTemplate> toAccountNumber = fundTransferTemplate.getToAccountOptions();
+
+        String[] fromAccountItems = new String[fromAccountNumber.size()];
+        String[] toAccountItems = new String[toAccountNumber.size()];
+
+        for (int i = 0; i < fromAccountNumber.size(); i++) {
+            fromAccountItems[i] = fromAccountNumber.get(i).getAccountNo();
+        }
+        ArrayAdapter<String> fromAccountAdapter = new ArrayAdapter<>(getActivity(), android.R.layout.simple_list_item_1, fromAccountItems);
+        spFromAccount.setAdapter(fromAccountAdapter);
+        spFromAccount.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long
+                    id) {
+                fromClientId = fundTransferTemplate.getFromAccountOptions().
```

Comment:
need to validate the lists for data. Also use the fromAccountNumber list that you have above.